Shocksupershaman - Copy/src/main/java/com/example/shocksupershaman/shocktask1/UserDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.example.shocksupershaman.shocktask1;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class UserDTO {
+    private String name;
+    private int age;
+
+    public UserDTO(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public static UserDTO of(String name, int age) {

```

Comment:
We don't need to use separate static methods. new UserDTO(name, age) is ok. As an alternative we can use lombok @Builder